packages/react/error-boundary/src/ErrorBoundary.tsx

```diff
@@ -95,3 +106,17 @@ export default class ErrorBoundary extends Component<PropsWithRef<PropsWithChild
     return children;
   }
 }
+
+const ErrorBoundary = forwardRef<{ reset?(): void }, ComponentProps<typeof BaseErrorBoundary>>((props, resetRef) => {
+  const { groupResetKey } = useErrorBoundaryGroup();
```

Comment:
Would exposing `groupResetKey` be necessary? While writing some tests, I thought only exposing the `reset` function might be nice.

```tsx
const reset = useResetErrorBoundaryGroup();

const group = useErrorBoundaryGroup();

group.reset(); // The implementation detail `groupResetKey` is private, it only exposes `reset`
```

Reply:
Thanks👍 I removed groupResetKey and renamed resetGroup as reset like your comment in [9a15657](https://github.com/toss/slash/commit/9a15657960255ee62abca60cae34ea1798124ac8)